src/main/resources/init_db.sql

```diff
@@ -7,3 +7,9 @@ CREATE TABLE `manufacturers` (
                                         `country` VARCHAR(225) NOT NULL,
                                         `is_deleted` TINYINT NOT NULL DEFAULT 0,
                                         PRIMARY KEY (`id`));
+CREATE TABLE `taxi`.`drivers` (
```

Comment:
before tables better to put somerthing like 

DROP TABLE IF EXISTS `manufacturers`;
DROP TABLE IF EXISTS `drivers`;
And what means `taxi`? There is another one schema?